src/Compilers/CSharp/Portable/Symbols/TypeParameterSymbol.cs

```diff
@@ -613,6 +625,11 @@ internal sealed override ObsoleteAttributeData ObsoleteAttributeData
         public abstract bool HasReferenceTypeConstraint { get; }
 
         // PROTOTYPE(NullableReferenceTypes): Should this API be exposed through ITypeParameterSymbol?
+        /// <summary>
+        /// Returns whether the reference type constraint (the 'class' constraint) should also be treated as nullable ('class?') or non-nullable (class!).
+        /// In some cases this aspect is unknown (null value is returned). For example, when 'class' constraint is specified in a NonNullTypes(false) context.  
```

Comment:
>In some cases this aspect is unknown (null value is returned). For example, when 'class' constraint is specified in a NonNullTypes(false) context. [](start = 12, length = 146)

Are there other cases? If not, perhaps: "Returns null when 'class' constraint is specified in a NonNullTypes(false) context."